### PR TITLE
Reorganizing and cleaning up repo

### PR DESCRIPTION
**Pull request by Dimitri-Dumont**
Mostly adding readme's to sub directories and removing a lot unused files.

### CODE REVIEW

File: API/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Launch backend server locally
+
+Open command line and cd to `API` folder create virtual env
+
+```$ python3 -m venv venv```
+
+Activate environment
+```source venv/bin/activate```
```
Comment (MythreyiT) [inline API/README.md:10]: Can we additionally add On Windows 10 command prompt it is: 
```venv/Scripts/activate```

File: README.md [hunk 2]
```diff
@@ -1,57 +1,16 @@
...
 
-## Super Cool Project
+## Brain Tumor Detection Project Overview
 
-### Contributors:
-Dimitri Dumont
-
-Gurleen Virk
-
-Mythreyi
```
Comment (MythreyiT) [inline README.md:10]: Please add full name : Mythreyi Thirumalai

Comment (MythreyiT) [reply README.md:10]: Do we want to keep TrainV2.py , Training.py and train.ipynb in the finalized one?

File: README.md [hunk 3]
```diff
@@ -1,57 +1,16 @@
...
-Once model training is complete we can have this image be labeled with predicted tumor and tumor location. 
+This project hopes to represents the possibilities in the application of AI for medical diagnostics, providing a robust tool for healthcare professionals to assist in the early detection and treatment of brain tumors.
 
+### Contributors:
+Dimitri Dumont
 
-![Alt Text](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExNWNlNnVhYzJveXY2djljeWV3NTcxeG5hZG5iNTExNmlkb3J5OGFmdyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/YbXLZ6dymH758xSEbM/giphy.gif) ![Alt Text](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExd2duNnlsazQ1bmpuaGUxcjR1amluZzNvc3FhNGp3eWZiYnVjNnlicyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/bJ4TVNYNUympPgcpem/giphy.gif)
+Gurleen Virk
 
+Mythreyi
```
Comment (MythreyiT) [inline README.md]: Please add Mythreyi Thirumalai

File: README.md (hunk 2 above)
Comment (MythreyiT) [reply README.md:10]: I don't see the requirements.txt being updated 
I think there were some libraries missing there - aggregating from Gurleen's code too. 
pillow
yaml
cv2
keras
flask
uvicorn

**Review by MythreyiT — CHANGES_REQUESTED** [review]
Hi Dimitri, 
Thank you for cleaning up the code , ready for it to be finalized. 
I have a couple of comments inline, nothing too major, please have a look. 

Do we want to keep both the previous Training.py and TrainV2.py ? How do we make it known that the code to be looked at is in train.ipynb?

**Comment by Dimitri-Dumont** [comment]
> Hi Dimitri, Thank you for cleaning up the code , ready for it to be finalized. I have a couple of comments inline, nothing too major, please have a look.
> 
> Do we want to keep both the previous Training.py and TrainV2.py ? How do we make it known that the code to be looked at is in train.ipynb?

I'm fine removing Training.py. Either TrainV2 or train are working files. I'll indicate that in the Model/README.md.

File: API/README.md (hunk 1 above)
Comment (Dimitri-Dumont) [reply API/README.md:10]: Sure, good catch!

File: README.md (hunk 2 above)
Comment (Dimitri-Dumont) [reply README.md:10]: Gotchya!

File: README.md (hunk 3 above)
Comment (Dimitri-Dumont) [reply README.md]: Okay!

**Review by MythreyiT — APPROVED** [review]
Sad to see my original code deleted - but TrainV2.py is essentially the same. 
I see you have changed the batch size from 32 to 8 and epochs from 50 to 1 in the TrainV2.py file , was this intentional?  I am finalizing the video ppt today and will be talking about the epochs and the batch size - will stick to the 32 and 50 in the ppt.

**Comment by Dimitri-Dumont** [comment]
> Sad to see my original code deleted - but TrainV2.py is essentially the same. I see you have changed the batch size from 32 to 8 and epochs from 50 to 1 in the TrainV2.py file , was this intentional? I am finalizing the video ppt today and will be talking about the epochs and the batch size - will stick to the 32 and 50 in the ppt.

I left it that way since this file functions as a proof of concept ability to run our training locally. The lower batch size requires less gpu memory and single epoch allows for training on a local machine in a reasonable amount of time. I feel it's fairly self evident in the code you can adjust the parameters.

**Pull request merged**